vignettes/tidy-evaluation.Rmd

```diff
@@ -115,7 +115,7 @@ positions. Therefore, `select()` understands column symbols like
 dplyr::select(mtcars, cyl)
 
 # Taking an unquoted symbol:
-var <- quote(cyl)
+var <- quo(cyl)
```

Comment:
This doesn't have to be a quosure since I'm referring to overscoped data (in this case a data frame column). I think @hadley only used `quo()` in the dplyr vignette to reduce the number of concepts. I think it's fine to leave it as a bare symbol here though.